### PR TITLE
AdsCert 2.0: Added logger bridge to adscert library

### PR DESCRIPTION
**Pull request by VeronikaSolovei9**

### CODE REVIEW

File: experiment/adscert/SignerLogger.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	"fmt"
+	"github.com/golang/glog"
+)
+
+type SignerLogger struct {
+}
+
+func (sl *SignerLogger) Debugf(format string, args ...interface{}) {
+	//there is no Debug level in glog
+	glog.Infof(format, args...)
```
Comment (mansinahar) [inline experiment/adscert/SignerLogger.go:13]: Is there any advantage of using `glog` vs the default level logger that comes with the `adscert/logger` package?

File: experiment/adscert/signer.go [hunk 2]
```diff
@@ -22,6 +23,7 @@ func (ns *NilSigner) Sign(destinationURL string, body []byte) (string, error) {
 }
 
 func NewAdCertsSigner(experimentAdCertsConfig config.ExperimentAdsCert) (Signer, error) {
+	logger.SetLoggerImpl(&SignerLogger{})
```
Comment (mansinahar) [inline experiment/adscert/signer.go:26]: I don't see the `logger` being used anywhere. Is there going to be a separate PR that makes use of this logger to log messages? Or am I missing something here?

**Review by mansinahar — APPROVED** [review]

**Review by AlexBVolcy — APPROVED** [review]
